testsuite/azure-arc-conformance.properties

```diff
@@ -6,6 +6,8 @@ az-storage-account=
 az-storage-account-sas=""
 # name of the partner offering; use this variable to distinguish between the results for different offerings
 offering-name=
+# Set below openshift variable to true when you are running openshift cluster  
```

Comment:
"Set the below variable to true when running the test suite on an openshift cluster

Reply:
Updated code comment and checked in the code.